server/store.go

```diff
@@ -26,7 +26,7 @@ type ResourceKind int
 
 const (
 	leaderKind ResourceKind = iota + 1
-	storageKind
+	regionKind
```

Comment:
Yep, I think about that too, but if we want to use peer to represent region's peer consistently, we need to change a lot of code, including some protobuf fields. So I just use region here, but if we really want to replace all "region" with "peer", we can do it in later.